knitr/pool-binary-trials/pool-no-pool.Rmd

```diff
@@ -869,50 +964,62 @@ random-walk Metropolis samplers (Papaspiliopoulos et al. 2003).  This
 basically amounts to changing the parameterization over which sampling
 is done, taking now a standard unit normal prior for a new variable,
 
-\[
+$$
 \alpha^{\mathrm{std}}_n = \frac{\alpha_n - \mu}{\sigma}.
-\]
+$$
 
 Then we can parameterize in terms of $\alpha^{\mathrm{std}}$, which
 has a standard-normal distribution
 
-\[
+$$
 p(\alpha^{\mathrm{std}}_n) = \mathsf{Normal}(\alpha^{\mathrm{std}}_n \, | \, 0, 1).
-\]
+$$
 
 We can then define our original $\alpha$ as a derived quantity
 
-\[
+$$
 \alpha_n = \mu + \sigma \, \alpha^{\mathrm{std}}_n.
-\]
+$$
 
-We code this implicitly in the Stan model by defining the
-likelihood as
+This decouples the sampling distribution
+for $\alpha^{\mathrm{std}}$ from $\mu$ and $\sigma$, greatly reducing
+their correlation in the posterior.  
 
-\[
-p(y_n \, | \, \alpha^{\mathrm{std}}_n, \mu, \sigma, K)
-\ = \
-\mathsf{BinomialLogit}(K_n, \ \mu + \sigma \, \alpha_n).
-\]
+Prior to Stan 2.19, a Stan implementation directly encoded the math.
```

Comment:
```suggestion
Prior to Stan 2.19, a Stan implementation directly encoded the math needed to reparameterize.
```

Reply:
added more prose - hoping to make the use of this clearer.